spec/features/tools/delete_tool_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+describe 'Delete tool', reset_provider: true, js: true do
+  before :all do
+    @ingested_tool, _concept_response, _native_id_1 = publish_tool_draft
+
+    @ingested_tool_for_delete_messages, _concept_response, @native_id_2 = publish_tool_draft
+  end
+
+  after :all do
+    delete_response = cmr_client.delete_tool('MMT_2', @native_id_2, 'token')
```

Comment:
I think we should add a comment to remove this after the reset_provider ticket is resolved (can't recall the ticket number right now)

Reply:
CMR-6332

Reply:
Comment added